etc/example_reporting.yaml

```diff
@@ -50,3 +50,10 @@ rest_app:
         baseuri: http://clarity.com
         username: user
         password: pass
+
+    available_coverages: [1, 2, 5, 10, 20]
+    available_yields:  # keys are required yields, and values are associated required yield q30s
+        5: 4
+        10: 8
+        20: 16
+        40: 32
```

Comment:
Can you also add these entries in the [docker/reporting.yaml](https://github.com/EdinburghGenomics/Reporting-App/blob/master/docker/reporting.yaml) file.
I tried the validation and it fails the upload because the database side aggregation fails.